src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -118,6 +118,16 @@ const MyApp = () => {
 
 2. Share the same `MeetingManager` instance with multiple different `MeetingProvider`s.
 
+**IMPORTANT NOTE**
+This approach has limitation though. This should be used only in very specific cases where you want to share the meeting manager instance
```

Comment:
nit: I don't think you need though at the end :)

Reply:
Yeah makes sense, thanks. Have removed it and now it is: This approach has limitations.